Fix/353 remove dropdown toolbar

ektasawant opened the pull request:
<!-- If this pull request fixes an Issue, link it below. If not, you can remove the line below -->
Fixes [353](https://github.com/brightlayer-ui/react-component-library/issues/353) .

<!-- Include a bulleted list summarizing the main changes you have made in this PR -->
#### Changes proposed in this Pull Request:
- Remove dropdown toolbar
- Update changelog

jeffvg commented:
Should storybook run with these changes?....we are exporting dropdowntoolbar from components core index

daileytj commented:
I removed the `DropdownToolbar` export Jeff mentioned and updated the package version. Let's get an alpha published from the `mui-v5` branch after this gets merged so we can update [the showcase 2711 branch](https://github.com/brightlayer-ui/react-showcase-demo/tree/feature/2711-upgrate-mui-to-v5) to remove the dropdown toolbar component.

ektasawant commented:
> I removed the `DropdownToolbar` export Jeff mentioned and updated the package version. Let's get an alpha published from the `mui-v5` branch after this gets merged so we can update [the showcase 2711 branch](https://github.com/brightlayer-ui/react-showcase-demo/tree/feature/2711-upgrate-mui-to-v5) to remove the dropdown toolbar component.

Thanks Tom.
